consent-ws/src/main/java/org/broadinstitute/consent/http/resources/Resource.java

```diff
@@ -25,7 +26,13 @@ protected Logger logger() {
 
     protected Response createExceptionResponse(Exception e) {
         try {
-            return dispatch.get(e.getClass()).handle(e);
+            logger().warn("Returning error response to client: " + e.getMessage());
+            ExceptionHandler handler = dispatch.get(e.getClass());
+            if (handler != null) {
+                return handler.handle(e);
+            } else {
+                return Response.serverError().entity(new Error(e.getMessage(), Response.Status.INTERNAL_SERVER_ERROR.getStatusCode())).build();
+            }
```

Comment:
Added some logging here. During testing, some of the logged errors were pretty useless.